### PR TITLE
Get apache test passing under ubuntu

### PR DESCRIPTION
**Pull request by martijnvans**
Just needed a few tweaks to the expected logs regexes.

### CODE REVIEW

**Comment by martijnvans** [comment]
ops agent test flaked out with some unrelated errors. Will kick off a new run and hopefully won't see any flakes.

File: integration_test/third_party_apps_data/test_config.yaml [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+    platforms_to_skip:
+      - centos-7
+      - centos-8
+      - rocky-linux-8
+      - sles-12
+      - sles-15
+  cassandra:
     # Skip all platforms listed above except debian-10 for now.
     # New applications should use platforms_to_skip sparingly
     # if at all.
```
Comment (igorpeshansky) [inline integration_test/third_party_apps_data/test_config.yaml:13]: Does this comment belong on the first use of `platforms_to_skip` above?

File: integration_test/third_party_apps_data/test_config.yaml [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
   apache:
+    # TODO: reenable distros incrementally.
+    platforms_to_skip:
+      - centos-7
+      - centos-8
+      - rocky-linux-8
+      - sles-12
+      - sles-15
+  cassandra:
     # Skip all platforms listed above except debian-10 for now.
```
Comment (igorpeshansky) [inline integration_test/third_party_apps_data/test_config.yaml:11]: Does it make sense to add some variant of this comment before line 3?

Comment (martijnvans) [reply integration_test/third_party_apps_data/test_config.yaml]: moved comment

File: integration_test/third_party_apps_data/test_config.yaml (hunk 1 above)
Comment (martijnvans) [reply integration_test/third_party_apps_data/test_config.yaml]: sure, moved.

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**